src/controllers/articleController.js

```diff
@@ -1,8 +1,78 @@
+import Article from "../models/article";
+import { ArticleServices } from "../services/articleServices";
+
 export class ArticleController {
     // TODO Don't access database from this file you only needs
-    createArticle(req, res, next) { }
-    getAllArticles(req, res, next) { }
-    getArticle(req, res, next) { }
-    updateArticle(req, res, next) { }
-    deleteArticle(req, res, next) { }
-}
+    async createArticle(req, res, next) {
+        try {
+            const data = new Article({
+                title: req.body.title,
+                content: req.body.content,
+                photo: req.body.photo,
+            })
+            const article = await ArticleServices.createArticle(data)
+            res.send(article)
+        } catch (error) {
+            res.status(404)
+            res.send({ error: error.message })
```

Comment:
Replace this with `res.status(404).send({ error: error.message })` and do the same for other methods